kbase-extension/static/kbase/css/appCell.css

```diff
@@ -7,24 +7,30 @@
     font-family: 'Oxygen', sans-serif;
 }
 
-.kb-app-cell .kb-app-warning {
+.kb-app-cell .kb-app-warning,
```

Comment:
The majority of new content in this file comes from [editor cell css file](https://github.com/kbase/narrative/pull/1867/files#diff-2f6db36b5bb17c632260c0b5be1b263d3ca6b5c9d3081d87899340de480abc1f), and mimics the existing `.kb-app-cell` styles.